<!-- Begin Schedule Section -->
<section id="schedule" class="schedule">
    <div class="content-wrapper text-left">
        <div class="col-md-10 col-sm-10 col-lg-offset-2 col-md-offset-2 col-sm-offset-1">
            <h2>Program of RECOMB-Seq 2023</h2>

            <p class="lead"><a href="/posters/">List of posters for the poster sessions.</a></p>

<h3>April 14, 2023</h3>
            
<p>8:00am – 9:00am Registration</p>

<p>9:00am – 9:10am Welcome and opening remarks</p>

<h4>Session 1 Proceedings</h4>
<ul>
<li>9:10am – 9:45am Jessica Bonnie, Omar Ahmed and Ben Langmead. <em>DandD: Efficient Measurement of Sequence Growth and Similarity</em> [<a href="https://www.biorxiv.org/content/10.1101/2023.02.02.526837.abstract">preprint</a>]</li>
<li>9:45am – 10:20am Kuan-Hao Chao, Pei-Wei Chen, Sanjit A. Seshia and Ben Langmead. <em>WGT: Tools and Algorithms for Recognizing, Visualizing and Generating Wheeler Graphs</em> [<a href="https://www.biorxiv.org/content/10.1101/2022.10.15.512390.abstract">preprint</a>]</li>
<li>10:20am – 10:55am Ondrej Sladky, Pavel Vesely and Karel Brinda. <em>Masked Superstrings as a Unified Framework for Textual k-mer Set Representations</em> [<a href="https://www.biorxiv.org/content/10.1101/2023.02.01.526717.abstract">preprint</a>]</li>
</ul>

<p>11:05am – 11:25am Coffee break</p>
            
<h4>Session 2 Overlays</h4>
<ul>
<li>11:25am – 11:50am Meryem Banu Cavlak, Gagandeep Singh, Mohammed Alser, Can Firtina, Joël Lindegger, Mohammad Sadrosadati, Nika Mansouri Ghiasi, Can Alkan and Onur Mutlu. <em>TargetCall: Eliminating the Wasted Computation in Basecalling via Pre-Basecalling Filtering</em> [<a href="https://www.biorxiv.org/content/10.1101/2022.12.09.519749.abstract">preprint</a>]</li>
<li>11:50am – 12:05pm Joël Lindegger, Damla Senol Cali, Mohammed Alser, Juan Gómez-Luna, Nika Mansouri Ghiasi and Onur Mutlu. <em>Scrooge: A Fast and Memory-Frugal Genomic Sequence Aligner for CPUs, GPUs, and ASICs</em> [<a href="https://arxiv.org/abs/2208.09985">preprint</a>]</li>
</ul>

<h4>Session 3 Keynote</h4>            
<ul>
<li>1:45pm – 2:45pm Shilpa Garg
</ul>    
            
<h4>Session 4 Overlays</h4>
<ul>
<li>2:45pm – 3:10pm Chirag Jain. <em>Coverage-Preserving Sparsification of Overlap Graphs for Long-Read Assembly</em> [<a href="https://www.biorxiv.org/content/10.1101/2022.03.17.484715.abstract">preprint</a>]</li>
<li>3:10pm – 3:35pm Zhenmiao Zhang, Hongbo Wang, Chao Yang, Yufen Huang, Zhen Yue, Yang Chen, Lijuan Han, Aiping Lyu, Xiaodong Fang and Lu Zhang. <em>Exploring High-Quality Microbial Genomes by Assembly of Linked-Reads with High Barcode Specificity Using Deep Learning</em> [<a href="https://www.biorxiv.org/content/10.1101/2022.09.07.506963.abstract">preprint</a>]</li>
</ul>

<p>3:35am – 4:00am Coffee break</p>            

<h4>Session 5 Short talks</h4>
<ul>
<li>4:00pm – 4:10pm Armaghan Sarvar, Lauren Coombe, René Warren and Inanc Birol Genome misassembly detection using Stash: A data structure based on stochastic tile hashing</li>
<li>4:10pm – 4:20pm Dmitry Antipov, Shilpa Garg, Adam Phillippy and Sergey Koren Integrating Hi-C sequencing data in verkko for gapless haplotype-resolved assembly</li>
<li>4:20pm – 4:30pm Zhenxian Zheng, Junhao Su, Tak-Wah Lam and Ruibang Luo ClairS: Accurate haplotype-aware long-read somatic variant calling using deep learning-based synthetic data learning</li>
<li>4:30pm – 4:40pm Vikram Shivakumar, Omar Ahmed, Sam Kovaka, Mohsen Zakeri and Ben Langmead Sigmoni: efficient pangenome multi-classification of nanopore signal</li>
<li>4:40pm – 4:50pm Katharine Jenike, Sam Kovaka, Matthias Benoit, Srividya Ramakrishnan, Shujun Ou, James Saterlee, Stephan Hwang, Iacopo Gentile, Anat Hendelman, Michael Passalacqua, Xingang Wang, Michael Alonge, Hamsini Suresh, Ryan Santos, Blaine Fitzgerald, Gina Robitaille, Edeline Gagnon, Melissa Kramer, Sara Goodwin, W. Richard McCombie, Jaime Prohens, Tiina E. Särkinen, Amy Frary, Jesse Gillis, Joyce Van Eck, Ben Langmead, Zachary B. Lippman and Michael C. Schatz Panagram: Alignment-free and interactive pan-genome visualization</li>
</ul>
    
<p>5:00pm – 6:30pm Poster session</p>
            
            
<h3>April 15, 2023</h3>

<h4>Session 6 Proceedings</h4>           
<ul> 
<li>9:20am – 9:55am Yunfei Hu, Yuying Zhao, Curtis Schunk, Yingxiang Ma, Tyler Derr and Xin Maizie Zhou. <em>ADEPT: Autoencoder with Differentially Expressed Genes and Imputation for a Robust Spatial Transcriptomics Clustering</em></li>
<li>9:55am – 10:30am Noor Pratap Singh, Michael I Love and Rob Patro. <em>TreeTerminus - Creating Transcript Trees Using Inferential Replicate Counts</em> [<a href="https://www.biorxiv.org/content/10.1101/2022.11.01.514769.abstract">preprint</a>]</li>
<li>10:30am – 11:05am Camila Duitama González, Samarth Rangavittal, Riccardo Vicedomini, Hugues Richard and Rayan Chikhi. <em>aKmerBroom: Ancient Oral DNA Decontamination Using Bloom Filters on k-mer Sets</em></li>
</ul>

<p>11:05am – 11:25am Coffee break</p>

<h4>Session 7 Overlays</h4>
<ul>
<li>11:25am – 11:50am Fernando Hugo Cunha Dias, Manuel Caceres, Lucia Williams, Brendan Mumey and Alexandru I. Tomescu. <em>A Safety Framework for Flow Decomposition Problems via Integer Linear Programming</em> [<a href="https://arxiv.org/abs/2301.13245">preprint</a>]</li>
<li>11:50am – 12:15pm Joseph Estabrook, William Yashar, Hannah Holly, Julia Somers, Olga Nikolova, Ozgun Babur, Theodore Braun and Emek Demir. <em>Predicting Transcription Factor Activity Using Prior Biological Information</em> [<a href="https://www.biorxiv.org/content/10.1101/2022.12.16.520295.abstract">preprint</a>]</li>
</ul>
    
<h4>Session 8 Keynote</h4>            
<ul>
    <li>1:45pm – 2:45pm Gioele La Manno. <em>RNA Velocity from dynamical exploration to inference.</em>

</ul>            
            
<h4>Session 9 Overlays</h4>
<ul>
<li>2:45pm – 3:10pm Ali Tugrul Balci, Mark Maher Ebeid, Panayiotis Benos, Dennis Kostka and Maria Chikina. <em>An Intrinsically Interpretable Neural Network Architecture for Sequence to Function Learning</em> [<a href="https://www.biorxiv.org/content/10.1101/2023.01.25.525572.abstract">preprint</a>] </li>    
<li>3:10pm – 3:35pm Yun William Yu. <em>On Minimizers and Convolutional Filters: a Partial Justification for the Effectiveness of CNNs in Categorical Sequence Analysis</em> [<a href="https://arxiv.org/abs/2111.08452">preprint</a>]</li>
</ul>

<p>3:35am – 4:00am Coffee break</p>            
        
<h4>Session 10 Short talks</h4>
<ul>
<li>4:00pm – 4:10pm Alex Lederer, Lorenzo Talamanca, Colas Droin, Maxine Leonardi, Irina Khven, Hugo Carvalho, Felix Naef and Gioele La Manno A probabilistic framework for parametrizing RNA velocity fields with manifold-consistent cell cycle dynamics</li>
<li>4:10pm – 4:20pm Leah Weber, Derek Reiman, Mohammed El-Kebir and Aly Khan  TRIBAL: Tree inference of B cell clonal lineages</li>
<li>4:20pm – 4:30pm Ahmet Cemal Alıcıoğlu, Mahmud Sami Aydın and Can Alkan    Pairwise sequence alignment with block and character edit operations</li>
<li>4:30pm – 4:40pm Serghei Mangul, Mohammad Vahed, Nicholas Darci-Maher, Kerui Peng, Jaqueline Brito, JungHyun Jung, Anushka Rajesh, Andrew Smith, Reid F. Thompson, Casey Greene, Jonathan Jacobs, Dat Duong and Eleazar Eskin    A comprehensive analysis of the reusability of public omics data across 3.8 million research publications</li>
</ul>

<p>4:40pm - 5:00pm Closing remarks</p>
            
<p>5:00pm – 6:30pm Poster session</p>
                        
            
        </div>
    </div>
</section>
<!-- End Schedule Section -->
