<!-- Begin Schedule Section -->
<section id="schedule" class="schedule">
    <div class="content-wrapper text-left">
        <div class="col-md-10 col-sm-10 col-lg-offset-2 col-md-offset-2 col-sm-offset-1">
           <h2>The list of accepted papers</h2>

<p class="lead">The exact program will be determined at a later date. Below you can find the list of accepted papers in the Proceedings and Overlay tracks. You can still submit your work to the Poster/Short talk track until March 12, 2023.</p>

<p>Talks from the previous editions of RECOMB-seq can be found at our <a href="https://www.youtube.com/@recomb-seq">YouTube channel</a>.</p>

<h3>Proceedings track</h3>

<ul>
<li>Jessica Bonnie, Omar Ahmed and Ben Langmead. DandD: Efficient Measurement of Sequence Growth and Similarity</li>
<li>Kuan-Hao Chao, Pei-Wei Chen, Sanjit A. Seshia and Ben Langmead. WGT: Tools and Algorithms for Recognizing, Visualizing and Generating Wheeler Graphs</li>
<li>Camila Duitama González, Riccardo Vicedomini, Hugues Richard and Rayan Chikhi. aKmerBroom: Ancient Oral DNA Decontamination Using Bloom Filters on k-mer Sets</li>
<li>Yunfei Hu, Yuying Zhao, Curtis Schunk, Yingxiang Ma, Tyler Derr and Xin Maizie Zhou. ADEPT: Autoencoder with Differentially Expressed Genes and Imputation for a Robust Spatial Transcriptomics Clustering</li>
<li>Noor Pratap Singh, Michael I Love and Rob Patro. TreeTerminus - Creating Transcript Trees Using Inferential Replicate Counts</li>
<li>Ondrej Sladky, Pavel Vesely and Karel Brinda. Masked Superstrings as a Unified Framework for Textual k-mer Set Representations</li>
</ul>


<h3>Overlay track<h3>

<ul>
<li>Ali Tugrul Balci, Mark Maher Ebeid, Panayiotis Benos, Dennis Kostka and Maria Chikina. An Intrinsically Interpretable Neural Network Architecture for Sequence to Function Learning</li>
<li>Meryem Banu Cavlak, Gagandeep Singh, Mohammed Alser, Can Firtina, Joël Lindegger, Mohammad Sadrosadati, Nika Mansouri Ghiasi, Can Alkan and Onur Mutlu. TargetCall: Eliminating the Wasted Computation in Basecalling via Pre-Basecalling Filtering</li>
<li>Fernando Hugo Cunha Dias, Manuel Caceres, Lucia Williams, Brendan Mumey and Alexandru I. Tomescu. A Safety Framework for Flow Decomposition Problems via Integer Linear Programming</li>
<li>Joseph Estabrook, William Yashar, Hannah Holly, Julia Somers, Olga Nikolova, Ozgun Babur, Theodore Braun and Emek Demir. Predicting Transcription Factor Activity Using Prior Biological Information</li>
<li>Chirag Jain Coverage-preserving sparsification of overlap graphs for long-read assembly</li>
<li>Joël Lindegger, Damla Senol Cali, Mohammed Alser, Juan Gómez-Luna, Nika Mansouri Ghiasi and Onur Mutlu. Scrooge: A Fast and Memory-Frugal Genomic Sequence Aligner for CPUs, GPUs, and ASICs</li>
<li>Yun William Yu. On Minimizers and Convolutional Filters: a Partial Justification for the Effectiveness of CNNs in Categorical Sequence Analysis</li>
<li>Zhenmiao Zhang, Hongbo Wang, Chao Yang, Yufen Huang, Zhen Yue, Yang Chen, Lijuan Han, Aiping Lyu, Xiaodong Fang and Lu Zhang. Exploring High-Quality Microbial Henomes by Assembly of Linked-Reads with High Barcode Specificity Using Deep Learning</li>
</ul>
 
        </div>
    </div>
</section>
<!-- End Schedule Section -->
