<!-- Begin Schedule Section -->
<section id="schedule" class="schedule">
    <div class="content-wrapper text-left">
        <div class="col-md-10 col-sm-10 col-lg-offset-2 col-md-offset-2 col-sm-offset-1">
            <h2>Program of RECOMB-seq 2024</h2>
        </div>
    </div>

    <div class="content-wrapper text-left col-md-10 col-sm-10 col-lg-offset-2 col-md-offset-2 col-sm-offset-1">
<h4>Accepted Papers to the Proceedings Track</h4>
<ul>
<li>Qixiu Du, May Nee Poon, Xiaocheng Zeng, Pengcheng Zhang, Zheng Wei, Haochen Wang, Ye Wang, Lei Wei and Xiaowo Wang.  Synthetic promoter design in Escherichia coli based on multinomial diffusion model.</li>
<li>Mohsen Zakeri, Nathaniel Brown, Omar Ahmed, Travis Gagie and Ben Langmead.  Movi: a fast and cache-efficient full-text pangenome index.</li>
<li>Eddie Ferro, Marco Oliva, Travis Gagie and Christina Boucher.  Building a Pangenome Alignment Index via Recursive Prefix-Free Parsing.</li>
<li>Francisco Sena, Eliel Ingervo, Shahbaz Khan, Andrey Prjibelski, Sebastian Schmidt and Alexandru Tomescu.  Flowtigs: safety in flow decompositions for assembly graphs.</li>
<li>Victor Levallois, Pierre Peterlongo, Yoann Dufresne, Bertrand Le Gal and Francesco Andreace.  The Backpack Quotient Filter: a dynamic and space-efficient data structure for querying k-mers with abundance.</li>
<li>Alexander Sasse, Sara Mostafavi and Maria Chikina.  Quick and effective approximation of in silico saturation mutagenesis experiments with first-order Taylor expansion.</li>
</ul>

<h4>Accepted Papers to the Overlay Track</h4>
<ul>
<li>Ilia Minkin and Steven Salzberg.  Quality assessment of splice site annotation based on conservation across multiple species.</li>
<li>Yunfei Hu, Yikang Li, Manfei Xie, Mingxing Rao, Yichen Henry Liu and Xin Maizie Zhou.  MaskGraphene: Advancing joint embedding, clustering, and batch correction for spatial transcriptomics using graph-based self-supervised learning.</li>
<li>Tim Dunn, Justin Zook, James Holt and Satish Narayanasamy.  Jointly benchmarking small and structural variant calls with vcfdist.</li>
<li>Md Hasin Abrar and Paul Medvedev.  PLA-complexity of k-mer multisets.</li>
<li>Mir Henglin, Maryam Ghareghani, William Harvey, David Porubsky, Sergey Koren, Evan Eichler, Peter Ebert and Tobias Marschall.  Phasing Diploid Genome Assembly Graphs with Single-Cell Strand Sequencing.</li>
<li>Sergey Vilov and Matthias Heinig.  Investigating the performance of foundation models on human 3â€™UTR sequences.</li>
<li>Weiman Yuan, Can Luo, Yunfei Hu, Liting Zhang, Zihang Wen, Yichen Henry Liu, Xian Mallory and Xin Maizie Zhou.  CNVeil enables accurate and robust tumor subclone identification and copy number estimation from single cell sequencing data.
</ul>
</div>

</section>
<!-- End Schedule Section -->


