---
layout: post
title: Posters at RECOMB-seq 2023
isStaticPost: true
---

<h3>Poster session Day 1, April 14, 2023</h3>

<ul>
<li>Seq-1 Armaghan Sarvar, Lauren Coombe, René Warren and Inanc Birol Genome misassembly detection using Stash:  A data structure based on stochastic tile hashing</li>
<li>Seq-2 Vikram Shivakumar, Omar Ahmed, Sam Kovaka, Mohsen Zakeri and Ben Langmead Sigmoni: efficient pangenome multi-classification of nanopore signal</li>
<li>Seq-3 Damla Senol Cali, Konstantinos Kanellopoulos, Joel Lindegger, Zülal Bingöl, Gurpreet Singh Kalsi, Ziyi Zuo, Can Firtina, Meryem Banu Cavlak, Jeremie S. Kim, Nika Mansouri Ghiasi, Gagandeep Singh, Juan Gómez Luna, Nour Almadhoun Alserr, Mohammed Alser, Sreenivas Subramoney, Can Alkan, Saugata Ghose and Onur Mutlu SeGraM: A Universal Hardware Accelerator for Genomic Sequence-to-Graph and Sequence-to-Sequence Mapping</li>
<li>Seq-4 Parham Kazemi, Hamid Mohamadi, Justin Chu, Lauren Coombe, Rene L Warren and Inanc Birol ntHits: streaming through raw sequencing data to profile and filter k-mers with selected multiplicities</li>
<li>Seq-5 Ecem İlgün, Ömer Yavuz Öztürk, Klea Zambaku, Juan Gómez Luna, Mohammed Alser, Ricardo Roman-Brenes, Can Alkan and The Biopim Project Characterization of Alignment and Search Algorithms for Short Read, Long Read, and Graph Mappers</li>
<li>Seq-6 Dmitry Antipov, Shilpa Garg, Adam Phillippy and Sergey Koren Integrating Hi-C sequencing data in verkko for gapless haplotype-resolved assembly</li>
<li>Seq-7 Zhenxian Zheng, Junhao Su, Tak-Wah Lam and Ruibang Luo ClairS: Accurate haplotype-aware long-read somatic variant calling using deep learning-based synthetic data learning</li>
<li>Seq-8 Klea Zambaku, Ricardo Roman-Brenes, Ömer Yavuz Öztürk, Can Alkan and Inanç Birol Copy number estimation using Counting Bloom Filters in de novo assembled genomes</li>
<li>Seq-9 Katharine Jenike, Sam Kovaka, Matthias Benoit, Srividya Ramakrishnan, Shujun Ou, James Saterlee, Stephan Hwang, Iacopo Gentile, Anat Hendelman, Michael Passalacqua, Xingang Wang, Michael Alonge, Hamsini Suresh, Ryan Santos, Blaine Fitzgerald, Gina Robitaille, Edeline Gagnon, Melissa Kramer, Sara Goodwin, W. Richard McCombie, Jaime Prohens, Tiina E. Särkinen, Amy Frary, Jesse Gillis, Joyce Van Eck, Ben Langmead, Zachary B. Lippman and Michael C. Schatz Panagram: Alignment-free and interactive pan-genome visualization</li>
<li>Seq-10 Rene Warren, Michael J Allison, M. Louie Lopez, Neha Acharya-Patel, Lauren Coombe, Cecilia L. Yang, Caren C Helbing and Inanc Birol Unikseq: unique region identification in genome sequences using a k-mer approach, to empower environmental DNA assay designs and comparative genomics studies</li>
<li>Seq-11 Ghanshyam Chandra and Chirag Jain Minichain: A new method for pangenome graph construction</li>
<li>Seq-12 Anna Lisiecka and Norbert Dojer ALIBI2: improved linearization of pangenome graphs</li>
<li>Seq-13 Alexander Sweeten, Adam Phillippy and Michael Schatz Mod.Plot: A Rapid and Interactive Visualization of Tandem Repeats</li>
<li>Seq-14 Sam Kovaka, Paul W. Hook, Vikram Shivakumar, Katharine M. Jenike, Luke Morina, Roham Razaghi, Winston Timp and Michael C. Schatz Nanopore signal alignment, analysis, and visualization with Uncalled4</li>
<li>Seq-15 Sina Barazandeh, Mahumd Sami Aydin, Berke Ucar, Can Alkan and Inanc Birol Using Minimizer Interarrival Distances for Read-Until Human Read Detection from Blood Samples Sequenced by Oxford Nanopore</li>
<li>Seq-16 Karel Sedlar, Petra Polakovicova and Ralf Zimmer Improving Functional Annotation of Bacterial Genomes with COGtools</li>
<li>Seq-17 William Yashar, Garth Kong, Jake Vancampen, Brittany Curtiss, Daniel Coleman, Lucia Carbone, Galip Yardimci, Julia Maxson and Theodore Braun GoPeaks: histone modification peak calling for CUT&Tag</li>
</ul>

<h3>Poster session Day 2, April 15, 2023</h3>

<ul>
<li>Seq-18 Alex Lederer, Lorenzo Talamanca, Colas Droin, Maxine Leonardi, Irina Khven, Hugo Carvalho, Felix Naef and Gioele La Manno A probabilistic framework for parametrizing RNA velocity fields with manifold-consistent cell cycle dynamics</li>
<li>Seq-19 Serghei Mangul, Mohammad Vahed, Nicholas Darci-Maher, Kerui Peng, Jaqueline Brito, JungHyun Jung, Anushka Rajesh, Andrew Smith, Reid F. Thompson, Casey Greene, Jonathan Jacobs, Dat Duong and Eleazar Eskin A comprehensive analysis of the reusability of public omics data across 3.8 million research publications</li>
<li>Seq-20 Reza Abbasi-Asl Machine learning enabled pattern discovery in large-scale spatial gene expression datasets</li>
<li>Seq-21 Yu-Ning Huang, Anushka Rajesh, Ram Ayyala, Aditya Sarkar, Ruiwei Guo, Irina Nakashidze, Shirley Monge, Dottie Yu, Qiushi Peng, Grace Scheg, Khooshbu Kantibhai Patel, Tejasvene Ramesh, Anushka Yadav, Fangyun Liu, Jay Himanshu Mehta and Serghei Mangul The systematic assessment of completeness of public metadata accompanying omics studies</li>
<li>Seq-22 Aurora Maurizio, Anna Sofia Tascini and Marco Jacopo Morelli SurfR: Surfing the cells' surfaceome</li>
<li>Seq-23 Fangyun Liu, Brian Nadel, Pelin Icer Baykal and Serghei Mangul Evaluating the Robustness and Reproducibility of RNA-Seq Quantification Tools</li>
<li>Seq-24 Leah Weber, Derek Reiman, Mohammed El-Kebir and Aly Khan TRIBAL: Tree inference of B cell clonal lineages</li>
<li>Seq-25 Ram Ayyala, Dottie Yu, Sergey Knyazev and Serghei Mangul Rigorous Benchmarking of HLA Callers for RNA-seq Data</li>
<li>Seq-26 Serghei Mangul RNA-Seq-based methods are able to effectively capture the clonotypes and estimate the diversity of TCR repertoires in T cell rich tissues and certain repertoires</li>
<li>Seq-27 Yu-Ning Huang, Yiting Meng, Naresh Amrat Patel, Jay Himanshu Mehta, Brittney Hua, Marina Fayzullina, Houda Alachkar and Serghei Mangul Assessing the completeness of immunogenetics databases across diverse populations</li>
<li>Seq-28 Ahmet Cemal Alıcıoğlu, Mahmud Sami Aydın and Can Alkan Pairwise sequence alignment with block and character edit operations</li>
<li>Seq-29 Thomas Krannich, Stephan Fuchs and Sophie Köndgen Detection of large tandem duplications in HMPV isolates</li>
<li>Seq-30 Kamila Marszałek, Michał B. Kowalski, Andrzej Ossowski, Rafał Płoski, Renata Zbieć Piekarska, Paweł P. Łabaj and Wojciech Branicki Predicting the origin of soil samples - performance evaluation of a new targeted High-Throughput Sequencing metagenomic tool.</li>
<li>Seq-31 Viorel Munteanu, Khooshbu Kantibhai Patel, Nitesh Kumar Sharma, Sergey Knyazev and Serghei Mangul A rigorous benchmarking of methods for SARS-CoV-2 lineage abundance estimation in wastewater</li>
<li>Seq-32 Hamdiye Uzuner and Johannes Köster Orthanq: orthogonal evidence based haplotype quantification</li>
<li>Seq-33 Junhao Su, Tak-Wah Lam and Ruibang Luo ClusterV: Accurate Detection of HIV Quasispecies and Drug Resistance Mutations using ONT Sequencing Data</li>
</ul>
